plos_connector/plos_unittests.py

```diff
@@ -0,0 +1,96 @@
+import requests
+import requests_mock
+import unittest
+from main import *
+
+
+class TestPlos(unittest.TestCase):
+    def setUp(self):
+        """Overrides setUp from unittest to get dataset for unit testing"""
+
+        self.url = "https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0231476&type=printable"
+        self.filename = "test_filename.pdf"
+
+
+
+class UrlError(TestPlos):
+    def test_url(self):
+        """Checks if the URL is valid """
+        
+        # Provide invalid url
+        invalid_url = "https://journals.plos.org/invalid-url"
+
+        
+        response = download_journals(invalid_url, self.filename) 
+
+        # Check for status code
+        assert response.status_code == 200
```

Comment:
recommend using unittest assert (not sure if these are? ) So it will report how many passed? / skipped/ and failed

Reply:
Added unit test assert statements